README.md

```diff
@@ -1,13 +1,7 @@
 # Project Express API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+This Express application serves as an API for querying avocado sales data. It provides endpoints to retrieve data for all avocado sales, sales by region, and sales by specific date. Additionally, it offers a root endpoint "/" that returns documentation of the available endpoints.
 
 ## View it live
 
```

Comment:
Hi Yifan, 

How nice to review such a clean project! 

You have almost done all of the requirements for this project.

- Your API should have at least 3 routes. Try to push yourself to do more, though!✅
  - The endpoint "/" should return documentation of your API using e.g. [Express List Endpoints](https://www.npmjs.com/package/express-list-endpoints)✅
  - A minimum of one endpoint to return a **collection** of results (array of elements).✅
  - A minimum of one endpoint to return a **single** result (single element).❌
- Your API should be [RESTful](https://www.notion.so/23473abe980e40aaa932914751055d22?pvs=21).

So i guess you just have to toss in an endpoint to search on id and you are all good for this weeks assignement 👏🏼